contributing-guides/style-guide.md

```diff
@@ -287,41 +271,72 @@ Use backticks on the following:
 - Standard streams: `stdout`, `stdin`, `stderr`. **Do not** use the full names (e.g. standard output).
 - Compression algorithms, e.g. `zip`, `7z`, `xz`.
 
-## Descriptions
+## Heading
 
-### Imperative Mood
+### More information links
 
-- **All descriptions must be phrased in the imperative mood.**
-- This also applies to all translations by default unless otherwise specified in the language-specific section below.
+- On the `More information` link line, we prefer linking to the author's provided documentation of the command line reference or the man page. When not available, use <https://manned.org> as the default fallback for all platforms
+(except `osx` and BSD platforms other than FreeBSD).
+Alternatively, you can link to the author's website or a tutorial page if the command doesn't have a documentation page.
 
-If you are afraid the commands may differ between platforms or operating systems (e.g. Windows vs macOS), most [tldr pages clients](https://github.com/tldr-pages/tldr/wiki/tldr-pages-clients) will choose the most suitable version of the command.
+- For `osx`: Apple distributes the built-in man pages [in Xcode](https://developer.apple.com/documentation/os/reading_unix_manual_pages).
+For commands documented there, we recommend using <https://keith.github.io/xcode-man-pages/>, an HTML export of all Apple's man pages bundled with Xcode.
 
-In this case, the information of the Windows version of `cd` (stored in `pages/windows/cd.md`) will be displayed by default to Windows users, and a generic/common version (stored in `pages/common/cd.md`) will be displayed for Linux, macOS, and other platforms.
+> [!IMPORTANT]
+> All links must be enclosed inside angular brackets (`<` and `>`).
 
-When writing descriptions for command examples, **check for any grammatical errors**. `Go to the specified directory` is preferred instead of:
+- We prefer translations to use the more information link of the English page by default.
```

Comment:
> We don't generally support showing localized versions of the pages in a majority of cases as the translations a lot of times are out of sync with the English page.

My understanding: it's because the links can be outdated if they change, right?

If so, I agree this can be a problem. I prefer "It is suggested to", but I support making the argument clear in the style guide itself:

```suggestion
- It is suggested to use a more information link with English content in both translations and English pages. That's because the links can eventually change, but the translations are often out of sync with the English pages.
```

I've also thought that a link update can eventually come from a translation and propagate to the others using our script, so I reworded the first sentence, what do you think?